islutils/builders.h

```diff
@@ -89,6 +89,30 @@ ScheduleNodeBuilder set(std::vector<ScheduleNodeBuilder> &&children);
 
 ScheduleNodeBuilder subtree(isl::schedule_node node);
 
+template <typename T>
+std::vector<ScheduleNodeBuilder> sequenceTransform(T t) {
```

Comment:
Let's call this something like `varargToVector`, it is in no way specific to sequence.
A small comment as to why this works (brace-initialization interpreted as either copy-construction or initializer-list-construction) is welcome.